### ISSUE TITLE
StartPosition Glitch?

### ISSUE THREAD

**Issue by andyl** [liftbridge-io/liftbridge#33]
The StartPosition behavior is not what I expect.  What I see:

    EARLIEST behaves like LATEST
    NEW_ONLY behaves like EARLIEST
    LATEST behaves like NEW_ONLY

Here's my code (from https://github.com/andyl/liftbridge_ruby/blob/master/lb_client)
![image](https://user-images.githubusercontent.com/90030/45372779-a9a08980-b5a2-11e8-85bd-8166874e04ab.png)

Here's the output
![image](https://user-images.githubusercontent.com/90030/45372918-e8ceda80-b5a2-11e8-8862-9c578d3c4ad5.png)

Here's the Liftbridge debug log
![image](https://user-images.githubusercontent.com/90030/45372951-03a14f00-b5a3-11e8-8247-3e3ea570da05.png)

**Comment by tylertreat** [comment]
What commit of the server are you using? I recently changed the ordering of the `StartPosition` enum which might cause this problem.

**Comment by tylertreat** [comment]
Also, the server logs don't match your code. Note that the server logs show subscriptions with `StartPosition` `OFFSET`, `LATEST`, `NEW_ONLY`, and `OFFSET`. Your code shows:

```ruby
stream_base     = subscribe 
stream_earliest = subscribe(:EARLIEST)
stream_latest   = subscribe(:LATEST)
stream_new_only = subscribe(:NEW_ONLY)
```

So something is definitely off.

**Comment by andyl** [comment]
OK if you recently changed the `StartPosition` enum that would explain it.  I'm using the `go get` liftbridge server - `liftbridge --version` -> `0.0.1`.  I'm using this spec: https://github.com/liftbridge-io/liftbridge-grpc/blob/master/api.proto 

Thanks for your note - I'll keep my eye on this and adjust as things re-sync.

**Comment by tylertreat** [comment]
Yeah, sorry for that breaking change. I'm trying to keep breaking changes to a minimum but making necessary changes before a stable 1.0 release.
